### PR TITLE
Small update on how to get boot2docker working on OSX.

### PR DESCRIPTION
**Pull request by BramGruneir**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,18 @@ docker run -t -i -p 8080:8080 cockroachdb/cockroach shell
...
 ```
 
+If the docker command fails but Docker is installed, you probably need to initialize it.
+On OSX:
+```bash
+# Setup Boot2Docker. This should only need to be done once.
+boot2docker init
+# Start Boot2Docker. These commands will need to be run once per reboot.
+boot2docker start
+$(boot2docker shellinit)
```
Comment (bdarnell) [inline README.md]: This needs to be run once per shell, which could be more than once per reboot depending on your workflow.

File: README.md [hunk 2]
```diff
@@ -57,6 +57,18 @@ docker run -t -i -p 8080:8080 cockroachdb/cockroach shell
 # root@82cb657cdc42:/cockroach#
 ```
 
+If the docker command fails but Docker is installed, you probably need to initialize it.
```
Comment (bdarnell) [inline README.md]: It would be good to have the exact error message here to help people who are searching for it. Do you remember what it was?

**Comment by bdarnell** [comment]
LGTM

File: README.md (hunk 2 above)
Comment (BramGruneir) [inline README.md]: done.

File: README.md (hunk 1 above)
Comment (BramGruneir) [inline README.md]: fixed.

**Pull request merged**